tools/Makefile

```diff
@@ -41,3 +41,9 @@ new-release:
 update-version:
 	## When manually updating major version
 	./releaser/scripts/update-version.sh
+
+.PHONY: generate_mocks
+generate_mocks:	
```

Comment:
We need to add mocking to contributing guide

Reply:
Let's make follow up PR. This is large PR anyway

Reply:
ok